app/scripts/filters/resources.js

```diff
@@ -125,6 +125,50 @@ angular.module('openshiftConsole')
       return displayName;
     };
   })
+  .filter('canI', function(AuthorizationService) {
+    return function(resource, verb) {
+      return AuthorizationService.canI(resource, verb);
+    };
+  })
+  .filter('canIDoAny', function(canIFilter) {
+    var resourceRulesMap = {
```

Comment:
Seems like inconsistent capitalization and pluralization in the keys here would make it difficult to use